app/views/builds/index.html.erb

```diff
@@ -5,8 +5,22 @@
 <section class="tabs">
   <div class="row">
     <%= search_form do %>
-      <%= search_select :external, [["Yes", "true"], ["No", "false"]], size: 1 %>
+      <%= search_select :external_status, Job::VALID_STATUSES, size: 1, label: "Status" %>
```

Comment:
Calling it status when it's only external status could lead to confusion.
Either add a `help: 'External status'` or call it `Ext Status` or make the query correctly do the "job ? job.status : external_status" (hard)

Reply:
yeah, rails is not really my thing 😂